### PR TITLE
Improve TestOidcToken

### PR DESCRIPTION
**Pull request by ebyhr**
Remove redundant catch block and add Language annotation.

### CODE REVIEW

File: gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java [hunk 1]
```diff
@@ -13,46 +13,33 @@
...
-                    + "}";
-
-            oidcTokens = objectMapper.readValue(jsonStr, OidcTokens.class);
-        }
-        catch (JsonProcessingException ex) {
-            log.error(ex.getMessage());
-            fail();
-        }
+        @Language("JSON")
+        String jsonStr = "{\"id_token\" : \"ABC235234\", "
```
Comment (mosabua) [inline gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java:34]: We should clean this up to a multi line string

**Review by mosabua — DISMISSED** [review]

File: gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java (hunk 1 above)
Comment (ebyhr) [reply gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java:34]: I didn't use it intentionally as it inserts new lines - the value will be different from the original text.

Comment (mosabua) [reply gateway-ha/src/test/java/io/trino/gateway/ha/security/TestOidcToken.java]: Fair enough

**Review by mosabua — APPROVED** [review]
Looks good.

**Pull request merged**